CHANGELOG.md

```diff
@@ -1,5 +1,85 @@
 # Changelog
 
+## 0.5.0
+
+Since the launch of Python version, project, and tool management capabilities back in August, we've seen extraordinary adoption of uv. We've been iterating rapidly since then: adding new features, fixing bugs, and improving the user experience. Despite moving quickly, stability and compatibility has been a major focus — we've made thirty releases since our last breaking change. Consequently, we've accumulated various changes that improve correctness and user experience, but could break some workflows. This release contains those changes; we do not expect them to require significant changes to your workflows. Many of the changes here have been marked as breaking out of an abundance of caution.
+
+### Breaking
+
+- Discover and respect `.python-version` files in parent directories ([#6370](https://github.com/astral-sh/uv/pull/6370))
+
+  Previously, uv only read `.python-version` files from the working directory. The new behavior is better aligned with that of `pyenv`. uv will not search for `.python-version` files beyond project boundaries.
```

Comment:
I think the last sentence is confusing because it makes it sound like that's the new behavior. But the new behavior is finding in parents -- it just doesn't go past project boundaries.